backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/EmailValidationsController.java

```diff
@@ -43,24 +39,21 @@ public EmailValidationsController(UserService userService) {
 	}
 
 	@GetMapping
+	@ResponseStatus(HttpStatus.METHOD_NOT_ALLOWED)
 	@Operation(summary = "List all email validations for a user")
-	public CollectionModel<EmailValidationModel> getEmailValidationByUserId(
+	public ResponseEntity<Void> getEmailValidationByUserId(
```

Comment:
Why not removing this GET endpoint? Wouldn't Spring handle it and returns 405?

Reply:
I agree and it was what I originally aimed to do, but the problem is the following to create a HATEOAS link:
https://github.com/DTS-STN/canadian-dental-care-plan/blob/58dd9075ded63fd4e165de224e94d4d6275e1263/backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/mapper/UserModelMapper.java#L47

Removing the endpoint would mean the mapper would have to `linkTo(..)` the POST endpoint - `verifyConfirmationCodeStatus(..)`.

The mapper would have to pass a `emailValidationModel`, and a `bindingResult` argument to be able to link to this method and handle the `BindException`, which was really ugly IMO.

Reply:
HATEOAS.....